pubsub/connection.go

```diff
@@ -100,14 +100,14 @@ func (c *Connection) readPump() {
 			}
 			break
 		}
-		if c.s.rCallback != nil {
+		if c.s.callback != nil {
```

Comment:
I think this is still "correct" but I don't think there should be a case where the callback is empty (why would we even run the server then)